transport-classes-io_uring/src/main/java/io/netty/channel/uring/IoUringBufferRing.java

```diff
@@ -55,21 +52,6 @@ void fill() {
         }
     }
 
-    boolean isExhausted() {
-        return !hasSpareBuffer;
-    }
-
-    /**
-     * Mark this buffer ring as exhausted. This should be done if a read failed because there were no more buffers left
-     * to use.
-     */
-    void markExhausted() {
```

Comment:
I removed these methods as how these worked in the past just doesn't work in practice to the nature of how multishot recv might triggered multiple failures.